src/main/java/uk/gov/justice/digital/hmpps/whereabouts/listeners/SqsDomainEventListener.kt

```diff
@@ -23,31 +23,32 @@ class SqsDomainEventListener(
   }
 
   init {
-    log.info("BVLS enabled=$bvlsEnabled for domain events")
+    log.info("Legacy BVLS enabled=$bvlsEnabled for domain events")
   }
 
   @SqsListener("domainevent", factory = "hmppsQueueContainerFactoryProxy")
   @WithSpan(value = "map-devs-hmpps_whereabouts_api_domain_queue", kind = SpanKind.SERVER)
   fun handleDomainEvents(requestJson: String?) {
-    if (bvlsEnabled) {
-      try {
-        log.info("Raw domain event message: {}", requestJson)
-        val (message, messageAttributes) = gson.fromJson(requestJson, Message::class.java)
-        val eventType = messageAttributes.eventType.value
-        log.info("Processing message of type {}", eventType)
+    try {
+      log.info("Raw domain event message: {}", requestJson)
+      val (message, messageAttributes) = gson.fromJson(requestJson, Message::class.java)
+      val eventType = messageAttributes.eventType.value
+      log.info("Processing message of type {}", eventType)
 
-        when (eventType) {
-          "prison-offender-events.prisoner.released" -> {
-            val transferEventMessage = gson.fromJson(message, ReleasedOffenderEventMessage::class.java)
+      when (eventType) {
+        "prison-offender-events.prisoner.released" -> {
+          val transferEventMessage = gson.fromJson(message, ReleasedOffenderEventMessage::class.java)
+
+          if (bvlsEnabled) {
             videoLinkBookingService.deleteAppointmentWhenTransferredOrReleased(transferEventMessage)
+          } else {
+            log.info("Legacy BVLS event processing is disabled, ignoring release event - {}", transferEventMessage)
```

Comment:
This is to cater for the feature switch being flipped.  We will log any potential events that may come in when we are running the migration.